filter.imp

```diff
@@ -1,38 +1,38 @@
-   { include: ["@[\"<]asm/page_types.h[\">]", private, "<asm/page.h>", "public"] },
-   { include: ["@[\"<]asm/pgtable_64_types.h[\">]", private, "<asm/pgtable.h>", "public"] },
-   { include: ["@[\"<]asm/pgtable_types.h[\">]", private, "<asm/pgtable.h>", "public"] },
-   { include: ["@[\"<]asm/page_64.h[\">]", private, "<asm/page.h>", "public"] },
-   { include: ["@[\"<]asm/page_32.h[\">]", private, "<asm/page.h>", "public"] },
-   { include: ["@[\"<]asm/page.h[\">]", "public", "<asm/page.h>", "public"] },
-   { include: ["@[\"<]asm/string_64.h[\">]", private, "<linux/string.h>", "public"] },
-   { include: ["@[\"<]asm-generic/errno-base.h[\">]", private, "<asm/errno.h>", "public"] },
-   { include: ["@[\"<]asm-generic/param.h[\">]", private, "<asm/param.h>", "public"] },
-   { include: ["@[\"<]asm(-generic)?/percpu.h[\">]", private, "<linux/percpu.h>", "public"] },
-   { include: ["@[\"<]asm(-generic)?/resource.h[\">]", private, "<linux/resource.h>", "public"] },
-   { include: ["@[\"<]asm(-generic)?/signal(-defs)?.h[\">]", private, "<linux/signal.h>", "public"] },
-   { include: ["@[\"<]asm-generic/delay.h[\">]", private, "<linux/delay.h>", "public"] },
-   { include: ["@[\"<]asm-generic/memory_model.h[\">]", private, "<asm/page.h>", "public"] }, 
-   { include: ["@[\">]asm-generic/access_ok.h[\">]", private, "<linux/uaccess.h>", "public"] }, 
-   { include: ["@[\"<]asm(-generic)?/uaccess.h[\">]", private, "<linux/uaccess.h>", "public"] },
-   { include: ["@[\"<]asm(-generic)?/kprobes.h[\">]", private, "<linux/kprobes.h>", "public"] },
-   { include: ["@[\"<]asm-generic/set_memory.h[\">]", private, "<asm/set_memory.h>", "public"] },
-   { include: ["@[\"<]asm-generic/sections.h[\">]", private, "<asm/sections.h>", "public"] },
-   { include: ["@[\"<]asm-generic/siginfo.h[\">]", private, "<asm/siginfo.h>", "public"] }, 
-   { include: ["@[\"<]asm-generic/div64.h[\">]", private, "<asm/div64.h>", "public"] }, 
-   { include: ["@[\"<]asm-generic/module.h[\">]", private, "<linux/module.h>", "public"] }, 
-   { include: ["@[\"<]asm-generic/fcntl.h[\">]", private, "<asm/fcntl.h>", "public"] },
-   { include: ["@[\"<]asm/pti.h[\">]", private, "<linux/pti.h>", "public"] },
-   { include: ["@[\"<]asm/pci.h[\">]", private, "<linux/pci.h>", "public"] }, 
-   { include: ["@[\"<]asm/ptrace.h[\">]", private, "<linux/ptrace.h>", "public"] }, 
-   { include: ["@[\"<]asm-generic/irq_regs.h[\">]", private, "<asm/irq_regs.h>", "public"] }, 
-   { include: ["@[\"<]asm(-generic)?/early_ioremap.h[\">]", private, "<linux/io.h>", "public"] },
-   { include: ["@[\"<]asm-generic/mman-common.h[\">]", private, "<asm/mman.h>", "public"] },
-   { include: ["@[\"<]asm-generic/mshyperv.h[\">]", private, "<asm/mshyperv.h>", "public"] },
-   { include: ["@[\"<]asm/tsc.h[\">]", private, "<linux/timex.h>", "public"] },
+   { include: ["@.asm-generic/unaligned.h.", private, "<asm/unaligned.h>", "public"] },
```

Comment:
`.` means "any single character, right?

I guess we'd want to match `<asm-generic/*>` or "asm-generic/*"?

What's the `@` symbol do in these regexes? Pulling up another table https://github.com/include-what-you-use/include-what-you-use/blob/26b48e7a24c933189a00c00f43fd9722e8c5d9a8/boost-1.75-all.imp for example, it's not obvious. Should we be using `@` at all? If so, why?  Maybe that is part of the issue?